### ISSUE TITLE
Can this method accelerate inference speed?

### ISSUE THREAD

**Issue by mynameischaos** [Zhouaojun/Incremental-Network-Quantization#8]

**Comment by Zhouaojun** [comment]
@mynameischaos  yes,  network quantization can accelerate  inference speed,  but  your hardware must  support low-precision bit shift， you can pay close attention to intel-altera and intel-movidious product.

**Comment by ouceduxzk** [comment]
I am curious what is the rough inference speedup on a intel cpu that support low precision bit shift, any numbers ?

**Comment by victorygogogo** [comment]
how to check the cpu support  "low-precision bit shift" ??@Zhouaojun
